src/Behat/Testwork/Output/Formatter.php

```diff
@@ -30,6 +30,13 @@
     public function getName();
 
     /**
+     * Returns file name.
+     * @param string $suite
+     * @return string
+     */
+    public function generateFilenameFor($suite);
```

Comment:
Adding new methods to existing interfaces would be a BC break. Besides that, `generateFilenameFor()` seems to be a very-specific behaviour for a particular formatter, not all formatters in general.